lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -230,6 +245,9 @@ export const migrateGranulesAndFiles = async (
     granulesResult: granuleMigrationResult,
     filesResult: fileMigrationResult,
   };
+  const filepath = 'granulesAndFilesMigrationErrorLog.json';
+  const errorFileWriteStream = fs.createWriteStream(filepath);
+  errorFileWriteStream.write('{ "errors": [\n');
```

Comment:
I'm on the fence, but I might abstract this logic into a `createErrorFileWriteStream` helper in the `storeErrors.ts` file. Just make it accept `migrationName` as an argument.

Reply:
https://github.com/nasa/cumulus/pull/2179/files#diff-56c5b1137921652643573bffebcd7fae07a919d6517811b49ab0f84759ba10beR15